cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeConfigurationBuilder.java

```diff
@@ -346,6 +352,48 @@ public KafkaBridgeConfigurationBuilder withHttp(KafkaBridgeHttpConfig http, Kafk
         return this;
     }
 
+    /**
+     * Configures the Strimzi Metrics Reporter. It is set only if user enables Strimzi Metrics Reporter.
+     *
+     * @param model     Strimzi Metrics Reporter configuration
+     *
+     * @return Returns the builder instance
+     */
+    public KafkaBridgeConfigurationBuilder withStrimziMetricsReporter(StrimziMetricsReporterModel model)   {
+        if (model != null) {
+            printSectionHeader("Strimzi Metrics Reporter configuration");
+            writer.println("bridge.metrics=" + TYPE_STRIMZI_METRICS_REPORTER);
+            // the kafka. prefix is required by the Bridge to pass Kafka client configurations
+            writer.println("kafka.metric.reporters=" + StrimziMetricsReporterConfig.KAFKA_CLASS);
+            writer.println("kafka." + StrimziMetricsReporterConfig.LISTENER_ENABLE + "=false");
+            writer.println("kafka." + StrimziMetricsReporterConfig.ALLOW_LIST + "=" + model.getAllowList());
+            writer.println();
+        }
+        return this;
+    }
+
+    /**
+     * Configures the JMX Prometheus Metrics Exporter.
+     *
+     * @param model     JMX Prometheus Metrics Exporter configuration
+     * @param isMetricsEnabled  Flag which indicates whether the metrics are enabled or not.
+     *
+     * @return Returns the builder instance
+     */
+    public KafkaBridgeConfigurationBuilder withJmxPrometheusExporter(JmxPrometheusExporterModel model, boolean isMetricsEnabled)  {
+        if (model != null || isMetricsEnabled) {
+            printSectionHeader("Prometheus Jmx Exporter configuration");
+            writer.println("bridge.metrics=" + JmxPrometheusExporterMetrics.TYPE_JMX_EXPORTER);
+           // if isMetricsEnabled is not used, we pass the path of the config file. If it is used, the bridge will use the fallback config
+            if (!isMetricsEnabled) {
+                writer.println("bridge.metrics.exporter.config.path="
+                        + KAFKA_BRIDGE_CONFIG_VOLUME_MOUNT + JmxPrometheusExporterModel.CONFIG_MAP_KEY);
+            }
+            writer.println();
+        }
+        return this;
+    }
```

Comment:
Some more spacing can improve the readabilkty a lot ... e.g. ....

```suggestion
    public KafkaBridgeConfigurationBuilder withJmxPrometheusExporter(JmxPrometheusExporterModel model, boolean isMetricsEnabled)  {
        if (model != null || isMetricsEnabled) {
            printSectionHeader("Prometheus Jmx Exporter configuration");
            writer.println("bridge.metrics=" + JmxPrometheusExporterMetrics.TYPE_JMX_EXPORTER);
            
           // if isMetricsEnabled is not used, we pass the path of the config file. If it is used, the bridge will use the fallback config
            if (!isMetricsEnabled) {
                writer.println("bridge.metrics.exporter.config.path="
                        + KAFKA_BRIDGE_CONFIG_VOLUME_MOUNT + JmxPrometheusExporterModel.CONFIG_MAP_KEY);
            }
            
            writer.println();
        }
        
        return this;
    }
```